plutus-tx-plugin/test/Plugin/Coverage/Spec.hs

```diff
@@ -36,11 +36,11 @@ boolTrueFalse = plc (Proxy @"boolTrueFalse") (\() -> True && False)
 boolOtherFunction :: CompiledCode (Maybe Integer -> Maybe Bool)
 boolOtherFunction = plc (Proxy @"boolOtherFunction") fun
 
-{-# INLINEABLE fun #-}
 fun :: Maybe Integer -> Maybe Bool
 fun x = case x of
   Just y | otherFun y -> Just False
   _                   -> Nothing
+{-# INLINEABLE fun #-}
```

Comment:
This change causes some of the tests to fail.  I suspect that the numbers in `coverage` need to be changed but I'm not quite sure what it does so I'll take a closer look tomorrow.